src/reactable/table.jsx

```diff
@@ -133,6 +133,36 @@ export class Table extends React.Component {
         this.tfoot = tfoot;
 
         this.initializeSorts(props);
+        this.initializeFilters(props);
+    }
+
+    initializeFilters() {
+        this._filterable = {};
+        // Transform filterable properties into a more friendly list
+        for (let i in this.props.filterable) {
```

Comment:
For some reason this is making tests fail? I have some theories why but I'm not quite sure